Validation/EventGenerator/src/DQMHelper.cc

```diff
@@ -0,0 +1,28 @@
+#include "Validation/EventGenerator/interface/DQMHelper.h"
+
+
+DQMHelper::DQMHelper(DQMStore::IBooker *i):ibooker(i){
+
+}
+
+DQMHelper::~DQMHelper(){
+
+}
+
+MonitorElement* DQMHelper::book1dHisto(const std::string &name,const std::string &title,int n,double xmin,double xmax){
```

Comment:
all these couts should be removed asap.
@inugent  please make a PR to fix this.